### PR TITLE
fix: issue with scrolling when topEl would be the cell instead of row

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes none

### CODE REVIEW

**Comment by g-cheishvili** [comment]
It still does the same

**Comment by droshev** [comment]
guys, can you summarize the issue and why we are fixing it?

**Comment by g-cheishvili** [comment]
> guys, can you summarize the issue and why we are fixing it?

Entire page scrolls when you navigate the focusable list. It is visible on core table focusable example

**Comment by droshev** [comment]
then we should have a bug reporting that. I would like to avoid getting PRs without real bugs, which revert after.

**Review by g-cheishvili — APPROVED** [review]
